src/components/ChecVerticalNavigation.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <nav class="vertical-navigation">
+    <ul
+      class="vertical-navigation__list"
+    >
+      <!--
+        @slot to display individual ChecVerticalNavItem component
+      -->
+      <slot />
+    </ul>
+  </nav>
+</template>
+
+<script>
+
+export default {
+  name: 'ChecVerticalNavigation',
+};
+</script>
+
+<style lang="scss">
+.vertical-navigation {
+  @apply rounded shadow-sm w-56 bg-white px-3 py-5;
```

Comment:
```suggestion
  @apply rounded-md shadow-sm bg-white px-3 py-5;
```

The removal of the `w-56` is something I think we should try to get in the habit of doing. The implementor should be able to choose width I think.